.github/workflows/gss.yml

```diff
@@ -0,0 +1,197 @@
+name: GraphScope Store CI
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+    paths:
+      - 'proto/**'
+      - 'interactive_engine/**'
+      - 'python/graphscope/client/**'
+      - 'charts/graphscope-store/**'
+      - '.github/workflows/gss-yml'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'proto/**'
+      - 'interactive_engine/**'
+      - 'python/graphscope/client/**'
+      - 'charts/graphscope-store/**'
+      - '.github/workflows/gss-yml'
```

Comment:
Ditto, add a path ignore to ignore md file

Reply:
done